exercises/J-array-get-set/exercises2.js

```diff
@@ -7,7 +7,7 @@
 */
 
 let numbers = [1, 2, 3]; // Don't change this array literal declaration
-
+numbers[3] = 4;
```

Comment:
nice! Usually it is a better practice to use [push](exercises/J-array-get-set/exercises2.js) to add a number at the end of an array. This way you don't need to know what is the last position of the array to add the number